.github/workflows/code-scanning.yml

```diff
@@ -0,0 +1,64 @@
+name: "code-scanning"
+
+on:
+  push:
+  pull_request:
+  # Schedule code scanning
```

Comment:
To be removed?

Reply:
Hum I actually want to keep this comment around. May be useful to set a cron job to scan the code at given intervals in addition to scan in `push/pull-request` (and I am sure I will forget that we can schedule cron jobs in workflows if I remove this comment)